resource/opentelemetry-resource-detector-azure-app-service/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) Microsoft Corporation.
```

Comment:
Should this be an Apache license? (ditto the MIT license headers below ;-)

Reply:
Yeah. Wasn't sure about this. This is the license we use on our side of OpenSource such as Azure-sdk. The resource detectors are the first examples I have seen so far of more client-specific code being stored under the OTel org. @ocelotl thoughts?

Reply:
Might be good to bring up during the SIG meeting.

Reply:
Put it on the calendar

Reply:
I'm no legal expert, and I think this could be a sensitive topic that may require the input from Microsoft legal team. I just found that there are at least 2 licenses known as the MIT license:

[Expat](https://www.gnu.org/licenses/license-list.html#Expat)
[X11](https://www.gnu.org/licenses/license-list.html#X11License)

They appear to be compatible with GNU GPL, and Apache 2.0 is also compatible with GNU GPL. Does that mean that we can use it? Maybe, not sure and better to check with an expert.

Reply:
Switched to apache 2